### ISSUE TITLE
[Accessibility]: Links/URLs in footer aren't passing contrast with background via Lighthouse test

### ISSUE THREAD

**Issue by AshleyHenry15** [posit-dev/product-doc-theme#115]
**Describe the accessibility issue**
A clear and concise description of what the accessibility is.

```
<div class="lh-audit__description" style="box-sizing: border-box; min-width: 0px; min-height: 0px; --inner-audit-padding-right: 10px; padding: 8px 10px 8px 26px; color: rgb(158, 158, 158); font-family: system-ui, Roboto, Helvetica, Arial, sans-serif; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(33, 33, 33); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><span style="box-sizing: border-box; min-width: 0px; min-height: 0px;">Low-contrast text is difficult or impossible for many users to read.<span> </span><a rel="noopener" target="_blank" href="https://dequeuniversity.com/rules/axe/4.10/color-contrast" style="box-sizing: border-box; min-width: 0px; min-height: 0px; color: rgb(144, 202, 249);">Learn how to provide sufficient color contrast</a>.</span></div><div class="lh-audit__stackpacks" style="box-sizing: border-box; min-width: 0px; min-height: 0px; color: rgb(245, 245, 245); font-family: system-ui, Roboto, Helvetica, Arial, sans-serif; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(33, 33, 33); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"></div>
Failing Elements
--
Downloads for version 2024.12.1 (Kousa Dogwood)<a href="https://dailies.rstudio.com/rstudio/kousa-dogwood/?_gl=1*bkd3ww*_ga*NzM2Nj…" data-original-href="https://dailies.rstudio.com/rstudio/kousa-dogwood/" target="_blank" rel="noopener" class="external">
Footnotes There are multiple distribution options, such as Rocky Linux, which …<div id="quarto-appendix" class="default">

Low-contrast text is difficult or impossible for many users to read. [Learn how to provide sufficient color contrast](https://dequeuniversity.com/rules/axe/4.10/color-contrast).
Failing Elements
Downloads for version 2024.12.1 (Kousa Dogwood)
<a href="https://dailies.rstudio.com/rstudio/kousa-dogwood/?_gl=1*bkd3ww*_ga*NzM2Nj…" data-original-href="https://dailies.rstudio.com/rstudio/kousa-dogwood/" target="_blank" rel="noopener" class="external">
Footnotes There are multiple distribution options, such as Rocky Linux, which …
<div id="quarto-appendix" class="default">
```

<img width="648" alt="Image" src="https://github.com/user-attachments/assets/e5bb8e59-fd18-4508-8e34-9139e63aee92" />

**Resolution**
If applicable, add the solution to resolve this accessibility issue.

Change color or increase URL font weight?

**Additional context**
